packages/contracts-watr/contracts/common/ProxyStorage.sol

```diff
@@ -61,6 +61,10 @@ contract ProxyStorage {
     address public chainReserveFeed;
     bool public proofOfReserveEnabled;
 
+    // XC20Wrapper variables
+    address public nativeToken;
+    mapping(address => mapping(address => uint256)) allowances;
```

Comment:
We could keep using the ERC20 `_allowances` field (L39).

Reply:
I updated it as you suggested, although since we're deploying the contracts to a new chain we could consider stripping the `ProxyStorage` of all the legacy stuff